packages/celotool/src/lib/attestation.ts

```diff
@@ -45,7 +45,7 @@ export async function requestMoreAttestations(
   txParams: CeloTransactionParams = {}
 ) {
   const unselectedRequest = await attestations.getUnselectedRequest(phoneNumber, account)
-  if (unselectedRequest.blockNumber === 0) {
+  if (unselectedRequest.blockNumber === 0 || (await attestations.isAttestationExpired(unselectedRequest.blockNumber))) {
```

Comment:
does `getActionableAttestations` do what you want?

Reply:
looks like it would, but I think keeping the flow in the bot the same as in Valora probably makes sense (Valora does the same right now, but had the expiration check)